psd-web/spec/features/team_spec.rb

```diff
@@ -34,12 +34,30 @@ def resend_link_selector(email)
       end
     end
 
-    context "as a normal user" do
-      let(:user) { create(:user, :activated, :psd_user, teams: [team]) }
+    context "as a team admin" do
+      let(:user) { create(:user, :activated, :team_admin, teams: [team]) }
 
-      scenario "does not display the link for any users" do
-        expect(page).not_to have_css("a[href*=\"#{resend_invitation_team_path(team)}\"]")
+      scenario "inviting an existing user shows an error message" do
+        click_link "Invite a team member"
+        expect(page).to have_selector("h1", text: "Invite a team member")
```

Comment:
```suggestion
        expect(page).to have_css("h1", text: "Invite a team member")
```
we use `have_css` (clearer than `have_selector`) everywhere let's keep consistency